src/bitcoin-cli.cpp

```diff
@@ -309,8 +309,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
         }
         return UNKNOWN_NETWORK;
     }
-    uint8_t m_details_level{0};      //!< Optional user-supplied arg to set dashboard details level
-    bool m_is_help_requested{false}; //!< Optional user-supplied arg to print help documentation
```

Comment:
Why is this bool removed? What was it doing before such that it is no longer needed now?

Reply:
Yes, it is no longer needed. Before, `m_is_help_requested` stored if the user passed "help" between `PrepareRequest()` and `ProcessReply()` inside the `NetinfoRequestHandler` class. Now, that logic happens outside of the class at line ~869.